scenario/V2Scenario.ts

```diff
@@ -0,0 +1,82 @@
+import { scenario } from './context/CometContext';
+import { exp } from '../test/helpers';
+import { expect } from 'chai';
+
+// note: meant for sanity checking v2 proposals, can normally be set to skip
+//  enable to check specific v2 actions with addresses hard-coded by network
+//  addresses to check and markets will need to be updated between runs
+
+scenario.skip(
+  'Compound v2 > allows a user to repay, borrow, repay cETH',
+  {},
+  async ({ actors }, context, world) => {
+    const { albert, betty } = actors; albert; betty;
```

Comment:
just curious, what does appending `albert; better;` to this line do?

isn't this the same as just:

```
const { albert, betty } = actors;
```

Reply:
Whoops good catch!

Reply:
Actually these were to quiet linter; but deleted the actors altogether now